static/js/tools/scatter/plot_options.tsx

```diff
@@ -86,7 +86,14 @@ function PlotOptions(): JSX.Element {
                     id="per-capita-y"
                     type="checkbox"
                     checked={y.value.perCapita}
-                    onChange={(e) => y.setPerCapita(e.target.checked)}
+                    onChange={(e) => {
+                      y.setPerCapita(e.target.checked);
+                      if (!y.value.perCapita && window && window.gtag) {
+                        window.gtag("event", "tool_chart_option_click", {
+                          tool_chart_option: "per capita",
```

Comment:
It just occured to me - do we need to specify what tool this is click is generated from?

Reply:
We have page path as a default parameter for every event. We can see which tool this click is generated from from the page path. e.g. /tools/timeline/